JSTests/stress/math-random-inline-vs-non-inline-paths.js

```diff
@@ -0,0 +1,24 @@
+//@ runDefault("--useJIT=1")
```

Comment:
If this test case reproduces without any extra flags, this line should be removed. Confirm that the test fails without your patch, and succeeds with it:

```
run-javascriptcore-tests --debug --no-build --jsc-only --no-fail-fast --memory-limited --jsc-stress --filter "random-inline"
```